### ISSUE TITLE
Q: there is a why to download subtitles [hebrew..]

### ISSUE THREAD

**Issue by JewSdarot** [asciidisco/plugin.video.netflix#95]
hello i am from israel and i want to watch movies offline on my tv
well i need to download the subtitles from netflix but the subtitle is images 
http://prntscr.com/g8o6m3

i see the update with support in hebrew subtitles 
so i think you can give me some info how to get the subtitles 

i know is not issues but you i think you can help me if you want :) 
have a great day and sorry for my english

**Comment by asciidisco** [comment]
No need to excuse :)
Image subtitles aren't possible at the moment, so Hebrew doesn't work yet.
We need to see if we can somehow integrate this into Kodi (I believe that this needs some core changes)

**Comment by JewSdarot** [comment]
ok.. thanks you for the help :)

**Comment by ymca11** [comment]
Hi asciidisco, why there's no plugin.video.netflix thread on Kodi forum?
It's not like it's illegal?! Just users with Netflix account can use it.

I have posted the problem with the subtitle there and someone commented that he didn't hear about it.
https://forum.kodi.tv/showthread.php?tid=322362&pid=2654551#pid2654551

**Comment by nemphys** [comment]
Any news on this one? It is a real bummer not to have subtitle support for all languages, since the module is otherwise working perfectly!
Judging from the replies on the thread posted above, it looks like native kodi support is there already (?) and it's a matter of the addon to support it.
Could this be the case?

**Comment by spokV** [comment]
Hi. Same here. When do you think it will be fixed?

**Comment by gergeo86** [comment]
please add greek subs.please

**Comment by miagg** [comment]
Another vote for IDX subs support on Kodi, although I'm pretty sure this is Kodi related and not plugin related.

**Comment by adno** [comment]
Would be great for me too, as Japanese subs are also images.

**Comment by vdnick** [comment]
Hi, also romanian subtitles are images.

**Comment by gergeo86** [comment]
please make greek subs appear .pleaseeeeeee

**Comment by Haris1977** [comment]
Just saw this thread. Yes we need Greek subs plz. Have opened a thread here also https://github.com/asciidisco/plugin.video.netflix/issues/361

**Comment by nemphys** [comment]
BTW, I recently noticed that Netflix must have changed the way greek subtitles are served in the newest content (images -> normal text), which makes them appear/work properly.
Check out 3% S02 and you will be delighted :-)

**Comment by Haris1977** [comment]
3% S02? What do y mean by that?

I just cant see greek subs in this addon though..

**Comment by nemphys** [comment]
I mean the TV series called "3%", 2nd season (eg. episode 1). You will see that greek subs are available (even with this plugin) and work fine, because Netflix changed them to be text instead of images.

**Comment by Haris1977** [comment]
Hm. I just checked it. You are right.:) But the problem remains with ALL the rest series/movies that are on greek netflix's database. Will they change ALL image subs to txt ? Cause i dont thing so..:(

**Comment by nemphys** [comment]
Yes, it's a bummer. I don't think image subs will ever be implemented, though, so don't get your hopes up. This is a very old issue and does not seem easy to solve by any means.
At least we will have greek subs working from now on :-)

**Comment by Haris1977** [comment]
Well thanks. So our only hopes - i guess - lie in @asciidisco to find a way to implement image based subs to his plugin. Keeping our fingers crossed..

**Comment by Haris1977** [comment]
I ve found a way to have ALL (text and image based) greek subs enabled but it s very buggy. See here for more info https://github.com/asciidisco/plugin.video.netflix/issues/361

**Comment by hobodrifterdavid** [comment]
Hello.
I think I have a solution for this issue.
Try adding 'webvtt-lssdh-ios8' to the list of profiles in manifest_request_data (MSL.py). Netflix should give provide you a manifest with a downloadable text-based XML file. You'll need to parse this into a suitable format for Kodi.

**Comment by Haris1977** [comment]
Plz check this version here https://www101.zippyshare.com/v/GGDe4CXN/file.html to see if greek subtitles do appear or not. I can't try it my own as at this time i do not have a netflix account

Thanks

**Comment by Essam315** [comment]
@Haris1977 
Hi, This file is not working anymore, use this new one, and the subtitles will work for you in Greek language for all the movies and TV shows.

https://github.com/asciidisco/plugin.video.netflix/archive/0.13.21.zip

**Comment by Essam315** [comment]
@hobodrifterdavid

Hi, 'webvtt-lssdh-ios8' already added to Netflix plugin and to inputstream.adaptive from version 0.13.16, and you can use the subtitles in all of the available languages. 

https://github.com/asciidisco/plugin.video.netflix/commit/58c85cf21fb8f1e2d1ec5738e6036643a22c3e01

**Comment by Haris1977** [comment]
Oh..:-o 

Does this mean that greek subs have been completely fixed for new and older tv shows?
